### PR TITLE
Code-Review additions to db models

### PR DESCRIPTION
**Pull request by mlam0527**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

_Your PR Notes Here_

### CODE REVIEW

File: client/components/cart-confirmation.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const Confirmation = props => {
+  return (
+    <div>
+      <h3>Pruchase is Processing</h3>
```
Comment (alexandralangton) [inline client/components/cart-confirmation.js:6]: typo in line 6 but we can fix that in another PR

**Review by alexandralangton — APPROVED** [review]
Looking good! One typo to fix (don't know how visible comments are so reiterating)

File: client/components/cart-confirmation.js (hunk 1 above)
Comment (mlam0527) [reply client/components/cart-confirmation.js]: can I change the code on here or do i have to redo PR?

**Review by mlam0527 — COMMENTED** [review]
I made a change in github with the three dots in the corner to fix typo

**Pull request merged**